### PR TITLE
AR Signpost

### PR DESCRIPTION
**Pull request by Laghzaouim**
This version contains:

- Table of Contents
- Introduction
- Context

The rest is still to be done but the structure of the blogpost is defined as mentioned in the table of contents.

### CODE REVIEW

File: _posts/2019-12-20-ar-signpost.md [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+<div style="text-align: center; margin: 0px auto;">
+ <a href="{{ '/img/2019-12-20-ar-signpost/ar-signpost-general-architecture.png' | prepend: site.baseurl }}" data-lightbox="ui" >
+<img alt="ikea space" src="{{ '/img/2019-12-20-ar-signpost/ar-signpost-general-architecture.png' | prepend: site.baseurl }}" class="image fit" style="margin:0px auto; max-width: 750px;">
+</a>
+</div>
+
+
+Let's put your favorite mall on the scene for a moment.
+In this mall you would find markers with QR codes strategically placed so that they are visible, and you have easy access to them.
+Each QR contains contextual information such as a unique identifier, the building in which it is located and the floor in which it is located.
```
Comment (Deryaa) [inline _posts/2019-12-20-ar-signpost.md]: Maybe: "Each QR contains contextual information such as a unique identifier and the building and floor in which it is located."

File: _posts/2019-12-20-ar-signpost.md [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+
+The AR scene starts from the point where the camera is located, which is usually [x=0, y=0, z=0] and the objects are located around it with different coordinates.
+In this case we are going to focus only on x and z-axis because y-axis represents the height of the objects so it will be constant.
+
+The points of interest we receive contain geographic coordinates (latitude and longitude) where they are located in the real world.
+The AR world does not understand geographic coordinates, so we have to convert these GPS coordinates to point x, z and y-axis in meters taking as reference the initial position of the AR camera.
+For this we use a technique called Web Mercator projection and in this way, we have our points of interest integrated into the real world as shown in the video above.
+
+Going back to our example before the mall, you wondered how this system would be implemented in a mall, didn't you? 
+Well there the web application intended for administrators or operators who are responsible for indicating where each point of interest is located in the building comes into play.
```
Comment (Deryaa) [inline _posts/2019-12-20-ar-signpost.md]: Well there, the web ... (comma makes it more readable I think)

Comment (Laghzaouim) [reply _posts/2019-12-20-ar-signpost.md]: > Great post! Just a few small remarks

Thank you very much for your feedback 🙂

**Review by bahadi7ibrahim — COMMENTED** [review]
please i need this code

**Pull request merged**